--- isdn4k-utils/FAQ/Makefile.in.target	2007-07-24 15:17:44.000000000 +0200
+++ isdn4k-utils/FAQ/Makefile.in	2007-07-24 15:42:45.000000000 +0200
@@ -125,7 +125,7 @@ $(DE_OUT_NAME).txt.gz: $(DE_OUT_TXT)
 
 # Install output files
 install: all
-	if [ -n "$(DESTDIR)$(TARGET_DIR)" ]; then \
+	if [ -n "$(TARGET_DIR)" ]; then \
 		$(INSTALL) -m 0755 -d $(DESTDIR)$(TARGET_DIR); \
 		$(INSTALL) -m 0644 $(SRC) $(DESTDIR)$(TARGET_DIR); \
 		$(INSTALL) -m 0644 $(OUT_TXT) $(DESTDIR)$(TARGET_DIR); \
