--- ./eicon/divautil/linload.c.str	2009-03-28 05:04:39.000000000 -0400
+++ ./eicon/divautil/linload.c	2009-03-28 05:05:16.000000000 -0400
@@ -123,13 +123,13 @@
 		strcat(filename, "/ds4bri.bit");
 
 		if ((fd = open(filename, O_RDONLY, 0)) == -1) {
-			sprintf(msg,dsp_name);
+			sprintf(msg,"%s",dsp_name);
 			(void)close(loadfd);
 			return(ERR_ETDD_DSP);
 		}
 
 		if (fstat(fd, &file_info)) {
-			sprintf(msg,dsp_name);
+			sprintf(msg,"%s",dsp_name);
 			(void)close(loadfd);
 			(void)close(fd);
 			return(ERR_ETDD_ACCESS);
@@ -205,13 +205,13 @@
 
 	/* open this DSP binary for reading */
 	if ((fd = open(filename, O_RDONLY, 0)) == -1) {
-		sprintf(msg,dsp_name);
+		sprintf(msg,"%s",dsp_name);
 		(void)close(loadfd);
 		return(ERR_ETDD_DSP);
 	}
 
 	if (fstat(fd, &file_info)) {
-		sprintf(msg,dsp_name);
+		sprintf(msg,"%s",dsp_name);
 		(void)close(loadfd);
 		(void)close(fd);
 		return(ERR_ETDD_ACCESS);
--- ./eicon/divautil/xlog.c.str	2009-03-28 05:04:12.000000000 -0400
+++ ./eicon/divautil/xlog.c	2009-03-28 05:04:25.000000000 -0400
@@ -594,7 +594,7 @@
   case 20:
     code = diva_word2le(buffer->info.l2.code);
     if ( ZERO_BASED_INDEX_VALID(code,l1_name) )
-      fprintf(Out, l1_name[code]);
+      fprintf(Out, "%s", l1_name[code]);
  else
       fprintf(Out, "UNKNOWN L1 STATE (0x%04x)", code);
     break;
--- ./eicon/eiconctrl.c.str	2009-03-28 05:03:41.000000000 -0400
+++ ./eicon/eiconctrl.c	2009-03-28 05:03:54.000000000 -0400
@@ -995,7 +995,7 @@
                         man_ent[i].Name,
                         AttSt,
                         man_ent[i].Var);
-                printf(MLine);
+                printf("%s",MLine);
         }
 }
 
--- ./eurofile/src/eft/tdu_log.c.str	2009-03-28 05:21:38.000000000 -0400
+++ ./eurofile/src/eft/tdu_log.c	2009-03-28 05:21:49.000000000 -0400
@@ -210,7 +210,7 @@
 
 void tdu_prompt(char * prompt)
 {
-	fprintf(stderr,prompt);
+	fprintf(stderr,"%s",prompt);
 	tdu_stderr_dirty = 0;
 }      
 
--- ./ipppd/main.c.str	2009-03-28 05:05:51.000000000 -0400
+++ ./ipppd/main.c	2009-03-28 05:06:05.000000000 -0400
@@ -240,7 +240,7 @@
         }
     }
     if (!ppp_ok) {
-        fprintf(stderr, no_ppp_msg);
+        fprintf(stderr, "%s",no_ppp_msg);
         exit(1);
     }
 
--- ./isdnlog/isdnlog/asn1_address.c.str	2009-03-28 05:00:58.000000000 -0400
+++ ./isdnlog/isdnlog/asn1_address.c	2009-03-28 05:02:20.000000000 -0400
@@ -160,7 +160,7 @@
 	XSEQUENCE_1(ParsePartyNumber, ASN1_NOT_TAGGED, ASN1_NOT_TAGGED, partyNumber);
 	XSEQUENCE_OPT_1(ParsePartySubaddress, ASN1_NOT_TAGGED, ASN1_NOT_TAGGED, partySubaddress);
 
-	str += sprintf(str, partyNumber);
+	str += sprintf(str, "%s", partyNumber);
 	if (strlen(partySubaddress))
 		str += sprintf(str, ".%s", partySubaddress);
 
@@ -191,13 +191,13 @@
 	
 	switch (publicTypeOfNumber) {
 	case 0: break; // unknown
-	case 1: str += sprintf(str, countryprefix); break;
-	case 2: str += sprintf(str, areaprefix); break;
+	case 1: str += sprintf(str, "%s", countryprefix); break;
+	case 2: str += sprintf(str, "%s", areaprefix); break;
 	case 3: str += sprintf(str, "(network)"); break;
 	case 4: str += sprintf(str, "(MSN)"); break;
 	case 6: str += sprintf(str, "(abbrev)"); break;
 	}
-	str += sprintf(str, numberDigits);
+	str += sprintf(str, "%s", numberDigits);
 
 	return p - beg;
 }
@@ -219,7 +219,7 @@
 	case 4: str += sprintf(str, "(local)"); break;
 	case 6: str += sprintf(str, "(abbrev)"); break;
 	}
-	str += sprintf(str, numberDigits);
+	str += sprintf(str, "%s", numberDigits);
 
 	return p - beg;
 }
--- ./isdnlog/isdnrep/isdnbill.c.str	2009-03-28 05:02:53.000000000 -0400
+++ ./isdnlog/isdnrep/isdnbill.c	2009-03-28 05:03:08.000000000 -0400
@@ -271,7 +271,7 @@
   memset(s, c, len);
   s[len] = 0;
 
-  printf(s);
+  printf("%s",s);
   printf("\n");
 } /* strich */
 
